ddtrace/internal/telemetry/writer.py

```diff
@@ -154,6 +162,13 @@ def add_event(self, payload, payload_type):
             }
             self._events_queue.append(event)
 
+    def add_error(self, code, msg):
+        # type: (int, str) -> None
+        """Add an error to be submitted with an event.
+        Note that this overwrites any previously set errors.
```

Comment:
Is overwriting existing errors the behavior that was defined in the instrumentation telemetry v2 spec? Can we just concatenate the error messages?

Reply:
I talked with @Kyle-Verhoog  about the worry of overwriting errors and it was decided that this usually won't happen, and that if it does it's currently acceptable within the telemetry spec we are following. We're not really implement telemetry v2 here, which calls for quite a few additional broader changes.

Reply:
Telemetry v2 work is scheduled for later.